build.gradle.kts

```diff
@@ -50,18 +50,29 @@ fun DependencyHandlerScope.includeApi(dependency: Any) {
     include(dependency)
 }
 
+fun DependencyHandlerScope.includeModApi(dependency: Provider<MinimalExternalModuleDependency>) {
+    include(dependency)
+    modApi(dependency) {
+        exclude(group = "net.fabricmc")
+    }
+}
+
 dependencies {
     includeApi(project(":core", configuration = "namedElements"))
     includeApi(project(":library", configuration = "namedElements"))
+
+    includeModApi(libs.serialization.minecraft)
+    includeApi(libs.collections.immutable)
+    includeApi(libs.klogging)
```

Comment:
Looking at how the project is structured, I think those additions would have to go either in the `core` or in the `library` subproject.

Reply:
I've read the project's README, and it seems like `core` is for the Kotlin adapter, and `library` is for FLK and other qkl specific things. So it made sense to us that each "sublibrary" in QKL should be declared in the root project. We could move that tho, not a problem.